### ISSUE TITLE
Implement pagination so we can import more than 75 bugs

### ISSUE THREAD

**Issue by fulldecent** [johnf/github-issue-importer#13]
Moving from the TODO file

**Comment by fulldecent** [comment]
To be clear, are you still accepting contributions to this project?? If so would a PR that fixes this be in scope to be merged?

**Comment by johnf** [comment]
Yes more than happy to accept pull requests or even hand the whole project over to someone else
